includes/fbbot/class-omise-fbbot-request-handler.php

```diff
@@ -0,0 +1,157 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if ( class_exists( 'Omise_FBBot_Request_Handler' ) ) {
+	return;
+}
+
+class Omise_FBBot_Request_Handler {
+	
+	private function __construct() {
+		// Hide the constructor
+	}
+
+	public static function handle_message_from( $sender_id, $user_message ) {
+		$message_store = Omise_FBBot_Message_Store;
```

Comment:
@igroomgrim better not to assign like this, if you wanna use dynamic variable for static class then can go with 
```php
$message_store = `Omise_FBBot_Message_Store`;

call_user_func( array( $message_store, 'check_greeting_words' ) );
```